client/src/layout/navbar-layout/NavbarLayout.tsx

```diff
@@ -30,28 +40,88 @@ const NavbarLayout: React.FC<INavbarLayout> = ({ collapse, setCollapse }) => {
                     className={styles.iconCollapse}
                     onClick={() => setCollapse(!collapse)}
                 />
-                <Typography.Title className={styles.logo}>
+                <Link to="./" className={styles.logo}>
                     9GAP
-                </Typography.Title>
+                </Link>
             </div>
             <div className={styles.itemContainer}>
-                <Button
-                    shape="circle"
-                    icon={<SearchOutlined />}
-                    className={styles.iconCollapse}
-                />
+                <Popover
+                    placement="bottom"
+                    trigger="click"
+                    content={
+                        <div className={styles.searchContainer}>
+                            <Input
+                                className={styles.searchInput}
+                                bordered={false}
+                                size="large"
+                                placeholder="Search..."
+                                onPressEnter={(e) => console.log(e.currentTarget.value)}
```

Comment:
Here if we need to wait for some module or api ready in order to implement this feature, we should add a comment for example: 
```
//TODO: Implement search feature when the api is ready 
onPressEnter={(e) => console.log(e.currentTarget.value)}
```